test/src/main/java/org/apache/accumulo/test/functional/OnDemandTabletUnloadingIT.java

```diff
@@ -169,6 +171,54 @@ public void testTabletUnloader() throws Exception {
     }
   }
 
+  /**
+   * Test the behavior of transitioning tablets from having a tablet availability of HOSTED to
+   * ONDEMAND. This transition should cause all tablets to unload because tablets are hosted but do
+   * not have a hosting requested column set.
+   */
+  @Test
+  public void testTransitionFromHostedToOndemand() throws Exception {
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+
+      String tableName = super.getUniqueNames(1)[0];
+
+      TreeSet<Text> splits = new TreeSet<>();
+      splits.add(new Text("f"));
+      splits.add(new Text("m"));
+      splits.add(new Text("t"));
+
+      NewTableConfiguration ntc = new NewTableConfiguration();
+      ntc.withSplits(splits);
```

Comment:
This can be made more succinct

```suggestion
      var splits = new TreeSet<>(Set.of(new Text("f"), new Text("m"), new Text("t")));
      var ntc = new NewTableConfiguration().withSplits(splits);
```

Also, I noticed that withSplits requires a sorted set. It really shouldn't. Returning a sorted set in an API makes sense, but requiring as input is too restrictive. It can sort internally, if needed. Of course, this wouldn't matter if Java had a `SortedSet.of()` immutable type.